features/dd-sdk-android-webview/src/main/kotlin/com/datadog/android/webview/internal/replay/WebViewReplayEventMapper.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.webview.internal.replay
+
+import com.datadog.android.webview.internal.rum.domain.RumContext
+import com.google.gson.JsonObject
+import java.lang.ClassCastException
+import java.lang.IllegalStateException
+import java.lang.NumberFormatException
+
+internal class WebViewReplayEventMapper(private val webViewId: Long) {
+
+    @Throws(
+        ClassCastException::class,
+        IllegalStateException::class,
+        NumberFormatException::class
+    )
+    fun mapEvent(
+        event: JsonObject,
+        rumContext: RumContext,
+        timeOffset: Long
+    ): JsonObject {
+        event.get("event")?.asJsonObject?.let { record ->
```

Comment:
@reviewers should we add here some telemetry ? it might be overkill but just wondering ?